ontology/yaml/resources/HVAC/entity_types/BLR.yaml

```diff
@@ -271,6 +271,15 @@ BLR_SS_SWTC_WDPC:
   - SS
   - SWTC
   - WDPC
+  
+BLR_SS_HTSWC_HTRWC:
```

Comment:
Can you clarify what you are representing here? A single boiler controls both to supply and return water temperatures? Or is this two boilers?

Reply:
apologies, added another type for the other boiler and removed return temp control form this one